modules/openy_features/openy_media/src/Plugin/Field/FieldFormatter/ImageLinkFormatter.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Drupal\openy_media\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Url;
+use Drupal\Core\Utility\LinkGeneratorInterface;
+use Drupal\image\Plugin\Field\FieldFormatter\ImageFormatterBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Plugin implementation of the image link field formatter.
+ *
+ * @FieldFormatter(
+ *   id = "openy_image_link",
+ *   label = @Translation("Image Link"),
+ *   field_types = {
+ *     "image"
+ *   }
+ * )
+ */
+class ImageLinkFormatter extends ImageFormatterBase implements ContainerFactoryPluginInterface {
```

Comment:
This class should live in `openy_media_image`, because it's being used only there.